docs/source/guide.rst

```diff
@@ -0,0 +1,54 @@
+User Guide
+==========
+
+This guide is a starting point for configuring and running the `test-operator`.
+
+Installation
+------------
+
+.. code-block:: bash
+
+    make install
```

Comment:
Do we want to add a step explaining how to log in to the cluster?

```
eval $(${HOME}/ci-framework-data/bin/crc oc-env)
export KUBECONFIG="${HOME}/.crc/machines/crc/kubeconfig"
oc login -u kubeadmin -p 12345678 https://api.crc.testing:6443/
export PATH=$PATH:/home/stack/bin
```

Reply:
I'm not sure that belongs to this documentation, on the other side, I don't see it mentioned anywhere else .. won't hurt I guess, at least for now until all other things settle down more

Reply:
I agree. It probably does not belong here but it might be good to have it here for now.